src/test/ensemble_tests.cpp

```diff
@@ -88,11 +89,11 @@ class EnsembleFlowTest : public ::testing::Test {
     }
 
     void performWrongPipelineConfigTest(const char* configFileContent) {
-        std::string fileToReload = "/tmp/ovms_config_file1.json";
+        std::string fileToReload = directoryPath + "/ovms_config_file1.json";
         createConfigFileWithContent(configFileContent, fileToReload);
         ConstructorEnabledModelManager managerWithDummyModel;
         managerWithDummyModel.startFromFile(fileToReload);
-        std::this_thread::sleep_for(std::chrono::milliseconds(1100));
+        std::this_thread::sleep_for(std::chrono::milliseconds(getConfigCheckTimePeriodDelayInMs(managerWithDummyModel)));
```

Comment:
Name `getConfigCheckTimePeriodDelayInMs` indicates it returns real wait time of model manager. We modify it for tests purpose. Name should indicate it is wait time enough for model manager to reload configuration from disk.